File: program/print-reverse-of-a-string/PrintReverseOfAString.cs
using System;  
  
namespace reverseString  
{  
    class Program  
    {  
        static void Main(string[] args)  
        {  
            string str = "", reverse = "";  
            int Length = 0;  
            Console.WriteLine("Enter a Word");  
            str = Console.ReadLine();  
            Length = str.Length - 1;  
            while(Length>=0)  
            {  
                reverse = reverse + str[Length];  
                Length--;  
            }  
            Console.WriteLine("Reverse word is : {0}",reverse);  
        }  
    }  
}  

File: program/program/print-reverse-of-a-string/PrintReverseOfAString.kt
fun main() {
    print("Enter String to Reverse: ")
    val str = readln()
    println()
    print(reverseString(str))
}

private fun reverseString(str: String): String {
    var reverse = ""
    for (i in str.lastIndex downTo 0) {
        reverse += str[i]
    }
    return reverse
}